src/taskgraph/decision.py

```diff
@@ -264,6 +272,38 @@ def _determine_more_accurate_base_ref(repo, candidate_base_ref, head_ref, base_r
     return base_ref
 
 
+def _determine_more_accurate_base_rev(repo, base_ref, candidate_base_rev, head_rev):
+    base_ref_or_rev = candidate_base_rev
+    if not candidate_base_rev:
+        logger.warning("base_rev is not set.")
+        base_ref_or_rev = base_ref
+    elif candidate_base_rev == Repository.NULL_REVISION:
+        logger.warning("base_rev equals the null revision. This branch is a new one.")
+        base_ref_or_rev = base_ref
+    elif not repo.does_revision_exist_locally(candidate_base_rev):
+        logger.warning(
+            "base_rev does not exist locally. It is likely because the branch was force-pushed. "
+            "Please set the _BASE_REV environment variable to let run-task fetch it."
```

Comment:
This message seems to imply an error, but looks like it will work just fine (which may cause some confusion). I also don't think the average developer will understand that they need to set `<REPO>_BASE_REV` in `.taskcluster.yml`. Can you make the instructions a bit more explicit?

Reply:
Great catch! I just changed the message. Let me know if looks clearer 🙂